docs/backup-restore.md

```diff
@@ -3,30 +3,63 @@
+Since Scalar DB server implements [scalar-admin](https://github.com/scalar-labs/scalar-admin) interface, you can easily pause the server to stop accepting incoming requests and drain in-flight requests.
+You can use paused duration and choose one of the following ways depending on the underlining database to make your databases transactionally-consistent even after restored.
+* Take backups of your databases during the paused duration.
+* Restore to a point in the paused duration with point-in-time-restore (PITR) feature.
+
+## How to create transactionally-consistent backup
+
+The easiest way to take a transactionally-consistent backup for Scalar DB on a non-transactional database is to use the [Scalar DB server](https://github.com/scalar-labs/scalardb/tree/master/server) or implement the [scalar-admin](https://github.com/scalar-labs/scalar-admin) interface properly in your application, you can easily pause the application without losing ongoing transactions.
+
+You must pause for 10 seconds to create a backup and use the midtime of the pause as a restore point.
+
+## Scalar DB Supported Databases
+
+### Cassandra
 
 Since Cassandra has a built-in replication mechanism, we don't always need a transactionally-consistent backup to recover a Cassandra cluster.
 
 For example, if replication is properly set to 3 and only the data of one of the nodes in a cluster is lost, we don't need a transactionally-consistent backup because the node can be recovered with a normal (transactionally-inconsistent) snapshot and the repair mechanism.
 However, if the quorum of nodes of a cluster loses their data, we need a transactionally-consistent backup to restore the cluster to a certain transactionally-consistent point.
 
-The easiest way to take a transactionally-consistent backup for Scalar DB on Cassandra is to stop a cluster, take the snapshots of all the nodes of the cluster, and start the cluster. If you implement [scalar-admin](https://github.com/scalar-labs/scalar-admin) interface properly in your application, you can easily pause the application without losing on-going transactions.
+Follow [How to create transactionally-consistent backup](#how-to-create-transactionally-consistent-backup) to create a transactionally-consistent backup.
+
+Another way to take a transactionally-consistent backup for Scalar DB on Cassandra is to stop a cluster, take the snapshots of all the nodes of the cluster, and start the cluster. 
 
 To minimize mistakes when doing backup and restore operations, it is recommended to use [Cassy](https://github.com/scalar-labs/cassy).
 Cassy is also integrated with `scalar-admin` so it can issue a pause request to the application of a Cassandra cluster.
 Please see [the doc](https://github.com/scalar-labs/cassy/blob/master/docs/getting-started.md#take-cluster-wide-consistent-backups) for more details.
 
-## JDBC databases
+### Cosmos DB
+
+You should create Cosmos DB account using `Continuous` backup policy to create continuous backups.
```

Comment:
Continuous backup is a must-have or nice-to-have?
It says `should` so it's nice to have?
Then, should it explain another way?

Reply:
Sorry for the inconvenience.
Document updated with the above suggestion.